lib/models/get-file-content.ts

```diff
@@ -6,9 +6,12 @@ export type GetFileContentOptionsType = { encoding: string | null } | null;
 /**
  * Return string if encoding is string and Buffer otherwise.
  */
-export type GetFileContentType = (filePath: string, options?: GetFileContentOptionsType) => Promise<string | Buffer>;
+export type GetFileContentType = (
+    normalizedRelativeFilePath: string,
```

Comment:
I like the changes of these parameter names. But should they also be updated in the readme? The readme only mentions:
`const getFileContent = async (filePath: string) => 'fileContent';`

It doesn't mention the options at all (which isn't that important I think). but the name could be updated to normalizedRelativeFilePath maybe?